packages/cli/test/fixtures/unit/commands/build/with-cron/vercel.json

```diff
@@ -1,10 +1,8 @@
 {
-  "functions": {
-    "api/overwrite/serverless.js": {
-      "cron": "0 10-20 * * *"
-    },
-    "api/overwrite/edge.js": {
-      "cron": "10 * * * *"
+  "crons": [
+    {
+      "path": "/api/cron-job",
```

Comment:
Does that path work with static files too, not just functions?

Reply:
Yes, it can be anything indeed.
This path is a free-form path. You can input whatever you want. We won't be checking if the path exists, for now at least.